DTS-STN/secure-client-hub
# Update auth helper functions to use getServerSession

### Changelog
fix:Update auth functions to use getServerSession instead of getSession

### Description of proposed changes:
This PR updates two of our Auth functions (`AuthIsValid` and `getLogoutURL`) to take in session as a parameter instead of accessing session information from the `getSession` function. The `getSession` function was never meant to be consumed server side even though it does technically work. Hoping this will cut down on some client fetch errors we're seeing in production.

### What to test for/How to test
1. Pull in branch
2. Type `npm run build` and `npm run start`
3. Ensure application runs as expected
